### ISSUE TITLE
Panel plugin with id agenty-flowcharting-panel could not be found

### ISSUE THREAD

**Issue by igorjasinski** [algenty/grafana-flowcharting#218]
Anyone help me please ?

i Have agenty-flowcharting installed running on grafana 6. But still got the message
"Panel plugin with id agenty-flowcharting-panel could not be found" when i try to create a dashboard.

Any thoughts?

**Comment by algenty** [comment]
see https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/
plugin is unsigned at this time

**Comment by Beanow** [comment]
My docker image `grafana/grafana:latest` was cached at v6.1.5.
Testing with: `GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=agenty-flowcharting-panel` [see config](https://grafana.com/docs/grafana/latest/administration/configuration/#allow_loading_unsigned_plugins) doesn't solve this issue.

Inspecting the network / debug tools it does load and execute `/public/plugins/agenty-flowcharting-panel/module.js`
Pausing on caught exceptions reveals a loading problem:
```
Error: Fetch error: 404 Not Found Instantiating http://localhost:8000/public/@grafana/data Loading http://localhost:8000/public/plugins/agenty-flowcharting-panel/module.js
b (webpack:///node_modules/systemjs/dist/system.js#4)
b (webpack:///node_modules/systemjs/dist/system.js#4)
a (webpack:///node_modules/systemjs/dist/system.js#4)
w (webpack:///node_modules/systemjs/dist/system.js#4)
w (webpack:///node_modules/systemjs/dist/system.js#4)
v.prototype[d.resolveInstantiate]/< (webpack:///node_modules/systemjs/dist/system.js#4)
d.resolveInstantiate (webpack:///node_modules/systemjs/dist/system.js#4)
import (webpack:///node_modules/systemjs/dist/system.js#4)
import (webpack:///node_modules/systemjs/dist/system.js#4)
import (webpack:///node_modules/systemjs/dist/system.js#4)
wl (webpack:///public/app/features/dashboard/state/PanelModel.ts#74)
loadPlugin (webpack:///public/app/core/components/PermissionList/PermissionListItem.tsx#28)
s (webpack:///node_modules/tslib/tslib.es6.js#97)
s (webpack:///node_modules/tslib/tslib.es6.js#78)
s (webpack:///node_modules/tslib/tslib.es6.js#71)
s (webpack:///node_modules/tslib/tslib.es6.js#67)
loadPlugin (webpack:///public/app/core/components/PermissionList/PermissionListItem.tsx#10)
onPluginTypeChanged (webpack:///public/app/features/dashboard/dashgrid/DashboardPanel.tsx#190)
onTypeChanged (webpack:///public/app/features/panel/GeneralTabCtrl.ts#21)
onClick (webpack:///public/app/features/dashboard/panel_editor/VisualizationTab.tsx#202)
f (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#14)
f (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#15)
E (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#15)
E (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#17)
A (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#18)
T (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#18)
O (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#21)
kn (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#84)
Rs (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#250)
Ne (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#39)
An (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js#86)
```

It suggests the plugin expects an external `@grafana/data` package to exist.
Looking at https://www.npmjs.com/package/@grafana/data it says this is in beta, and looking at the version history there isn't a 6.1.x version of it.

Next I tried `docker pull grafana/grafana` moving me up to v7.1.5 the plugin loads no problem.

So I'm expecting the `@grafana/data` dependency of the plugin gives it a minimum version requirement of somewhere around v6.4.x?

**Comment by xkilian** [comment]
Yeah, 6.4.x+. 6.4.1 doesn't work.

**Comment by algenty** [comment]
Fixed in 0.9.1 SNAPSHOT, i use too early grafana libs v7
